ibis/pandas/core.py

```diff
@@ -85,6 +85,49 @@
 :class:`~ibis.expr.operations.ScalarParameter`, in which case the ``scope``
 passed to ``post_execute`` would be the bound values passed in at the time the
 ``execute`` method was called.
+
+
+Scope
```

Comment:
Can you move these to scope.py as well?

Reply:
Actually, I think it's fine to leave as this because lot of these are related to pandas execution. But perhaps the parts about "get result smaller context if larger context is already evaluated" and also "how to merge scope with same op with smaller and large context" (in other words, scope/timecontext specific logic) to the scope module.

Reply:
I moved setting and getting logics to `scope.py` and leave high level statement about scop here.